superset-embedded-sdk/release-if-necessary.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const { execSync } = require('child_process');
+const { name, version } = require('./package.json');
+
+console.log(`checking if ${name}@${version} needs releasing`);
+
+// need to build and publish the package iif the current version does not exist on npm
```

Comment:
```suggestion
// need to build and publish the package if the current version does not exist on npm
```

Reply:
iif means "if and only if"

Reply:
oh actually I remembered wrong, it's "iff" lol

Reply:
```suggestion
// need to build and publish the package iff the current version does not exist on npm
```

Reply:
Can we write "if and only if" to avoid confusion? It was not clear for me at least 🤣